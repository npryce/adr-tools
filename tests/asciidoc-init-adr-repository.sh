adr init -t adoc
ls doc/adr
cat doc/adr/0001-record-architecture-decisions.adoc
