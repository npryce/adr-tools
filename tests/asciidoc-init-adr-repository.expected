adr init -t adoc
doc/adr/0001-record-architecture-decisions.adoc
ls doc/adr
0001-record-architecture-decisions.adoc
cat doc/adr/0001-record-architecture-decisions.adoc
= 1. Record architecture decisions

Date: 1992-01-12

== Status

Accepted

== Context

We need to record the architectural decisions made on this project.

== Decision

We will use Architecture Decision Records, as http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions[described by Michael Nygard].

== Consequences

See Michael Nygard's article, linked above. For a lightweight ADR toolset, see Nat Pryce's https://github.com/npryce/adr-tools[adr-tools].
